chsdi/static/doc/source/services/sdiservices.rst

```diff
@@ -949,6 +949,98 @@ Example
 - Check WMTS with Swiss ech-0056 profile (html): `https://api3.geo.admin.ch/owschecker/form?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS <../../../owschecker/form?base_url=http%3A%2F%2Ftile1-sitn.ne.ch%2Fmapproxy%2Fservice&service=WMTS>`_
 
 
+.. _terrain_service_description:
+
+---------------
+
+Terrain Service
+---------------
+
+A RESTFul implementation of "`Cesium <http://cesiumjs.org/>`_" `Quantized Mesh <https://cesiumjs.org/data-and-assets/terrain/formats/quantized-mesh-1.0.html>`_ terrain service.
```

Comment:
Official doc for quantized mesh has move here: https://github.com/AnalyticalGraphicsInc/quantized-mesh